addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/AutomationPanel.java

```diff
@@ -730,6 +751,13 @@ private void updateSaveButton(Event event, boolean enable) {
         }
     }
 
+    private void updateSaveAsButton(Event event, boolean enable) {
```

Comment:
This method should not be needed actually, we just want to change the enabled state when the plan changes which is already covered by `setCurrentPlan`.